src/Tribe/Settings_Helper.php

```diff
@@ -5,170 +5,159 @@
 	die( '-1' );
 }
 
-if ( class_exists( 'Tribe__Settings_Helper' ) ) {
```

Comment:
I didn't review this file, assuming you just wrapped in a `class_exists()` check without other changes... but the github diff viewer doesn't make that clear... You could choose to namespace the plugin instead or in addition to a class exists check.

Reply:
Yes. just added class_exists.